### ISSUE TITLE
Need correction wrong storage when deploy server

### ISSUE THREAD

**Issue by okamah** [UpCloudLtd/upcloud-whmcs-module#19]
Hi, 

I just download modul and activate in my backend.
when i trying to create, system execute storage only 10GB even though I have choose a plan, but it force it to 10Gb storage, when i check script in file manager.php I found this parameter 10 that trigger deployment with 10Gb:

--

public function createServer()
    {
        $zone = ($this->params['configoptions']['Location'] == '') ? $this->params['configoption1'] : $this->params['configoptions']['Location'];
        **$size = ($this->params['configoptions']['Storage'] == '') ? 10 : $this->params['configoptions']['Storage'];**

---

because my coding skill very bad, any body can help me to correction above variable $size? I just remove only 10, but error storage parameter invalid.

thank you

**Comment by wpexpertinbd** [comment]
if you are using "testing credit" then upgrade your account to normal  hope it will solve the issue.

**Comment by SPCnet** [comment]
Hi,

I have the same problem. I've created the service plans as described in the guide, where only Plan, Location and OS has to been selected. Every server created through the plugin have 10gb of storage, regardless of the selected plan.

My account is normal. I've made a one-time payment before start to use the WHMCS plugin.

![created](https://user-images.githubusercontent.com/12526444/108725644-e2403d80-7526-11eb-8376-70594a10672b.PNG)
![service](https://user-images.githubusercontent.com/12526444/108725660-e5d3c480-7526-11eb-9433-7afcb0f597ce.PNG)
